src/netlog/netlog-network.c

```diff
@@ -167,16 +171,27 @@ int manager_network_connect_socket(Manager *m) {
         if (r < 0)
                 return r;
 
-        log_debug("Connecting to remote server: '%s'", pretty);
+        switch (m->protocol) {
```

Comment:
we already doing it via `protocol_to_string`  no ?

Reply:
dito